package NYT.NHive.NProto;

import "yt/core/misc/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqAcknowledgeMessages
{
    required NYT.NProto.TGuid cell_id = 1;
    required int32 last_acknowledged_message_id = 2;
}

////////////////////////////////////////////////////////////////////////////////

message TReqUnregisterMailbox
{
    required NYT.NProto.TGuid cell_id = 1;
}

////////////////////////////////////////////////////////////////////////////////

